src/Site/Controller/Base.php

```diff
@@ -25,7 +25,7 @@ public function __construct() {
         $this->data['smallAvatarUrl'] = '';
         $this->data['userName'] = '';
         $this->data['version'] = VERSION;
-        $this->data['useMinifiedJs'] = USE_MINIFIED_JS;
+        $this->data['useMinifiedJs'] = false;
```

Comment:
Note you also need the `use Sil\PhpEnv\Env; // https://github.com/silinternational/php-env#class-env-summary-of-functions` at the top.  Github won't let me suggest that edit :(
I actually don't know if this is totally correct PHP syntax YMMV.
```suggestion
        $this->data['useMinifiedJs'] = (Env::requireEnv('ENVIRONMENT') == 'production');
```